packages/lib/file-api-driver-dropbox.js

```diff
@@ -171,24 +172,83 @@ class FileApiDriverDropbox {
 		if (typeof content === 'string') content = shim.Buffer.from(content, 'utf8');
 
 		try {
-			await this.api().exec(
-				'POST',
-				'files/upload',
-				content,
-				{
-					'Dropbox-API-Arg': JSON.stringify({
-						path: this.makePath_(path),
-						mode: 'overwrite',
-						mute: true, // Don't send a notification to user since there can be many of these updates
-					}),
-				},
-				options,
-			);
+			if (options && options.source === 'file') {
```

Comment:
I think with your changes you made every Dropbox request 3 times slower? Instead of having one request, you have three (start/append/complete) if I'm reading correctly.

Reply:
no, it didn't, instead of uploading a 10 MB attachment in one go, I first requested a `session_id` then I uploaded the file chunked. 

this happens **only** to attachments, not notes, folders, or locks.

Reply:
But 99.9% of attachment are below the limit so that's still a lot of complexity for something that could be uploaded in one request.

Reply:
I can write an additional `if` statement to only split files above a certain size into chunks while uploading, but the original issue addresses that attachments above 150 MB are not uploadable.